nvtabular/column_selector.py

```diff
@@ -13,16 +13,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-from dataclasses import dataclass, field
 from typing import List
 
 import nvtabular
 
 
-@dataclass
 class ColumnSelector:
-    _names: List[str]
-    subgroups: List["ColumnSelector"] = field(default_factory=list)
+    def __init__(self, names: List[str], subgroups: List["ColumnSelector"] = None):
+        self._names = names
```

Comment:
It doesn't need to be addressed here, but I've been wondering if we should also make `names` an optional parameter, so that creating a `ColumnSelector` with only subgroups is easier. Right now, you have to do `ColumnSelector([],subgroups=[a_subgroup])`, which seems unnecessarily verbose.

Reply:
interesting... I can add that in now. No biggie and I will add a test also.  That way we support both methods.